### ISSUE TITLE
Custom Role names are not inserted into ROLES table automatically

### ISSUE THREAD

**Issue by markuzzi** [FraunhoferIOSB/FROST-Server#2032]
If the role names for CRUD are customized e.g. using env vars, FROST still creates the standard roles in the ROLES table (read, create, delete, update, admin).

I would have expected that FROST might create the customized role names instead.

**Comment by hylkevds** [comment]
Good point.
Though that feature is for when an existing users/rights database exists, in which case FROST doesn't need to create any data in those tables. If you're depending on FROST to generate placeholder data, why change the roles names?

We'd need to look into how to best pass those role names from environment variables to Liquibase.

**Comment by markuzzi** [comment]
Got your point. We change them, because in our setup, multiple components are using these role names. With your answer, let me rephrase my initial comment: It would be convenient, if FROST could generate the customized roles instead of the standard roles :-)

**Comment by markuzzi** [comment]
We switched to FROST default roles. So, this is not very urgent for us anymore 👍
